client-src/elements/chromedash-feature-table.js

```diff
@@ -338,6 +355,45 @@ class ChromedashFeatureTable extends LitElement {
     return nothing;
   }
 
+  shiftStartBy(delta) {
+    const proposedStart = this.start + delta;
+    this.start = clamp(proposedStart, 0, this.totalCount - 1);
+    this.fetchFeatures();
+  }
+
+  renderPagination() {
+    const first = this.start + 1;
```

Comment:
Nit: `first` and `last` variable names can be more descriptive. `OfferLeft` and `offerRight` can be `hasPreviousPage` and `hasNextPage`

Reply:
Also why do we need to `+ 1` here?

Reply:
I renamed the variables.

The `+1` is to make the index one-based rather than zero-based.  Humans prefer to see one-based index numbers in the UI.   There is no `+ 1` for the last index because the index of the last element of an array is normally `length - 1`, so it would be `length -1 + 1`, so I just left those off.